ddtrace/ext/ci.py

```diff
@@ -68,25 +68,34 @@ def extract_appveyor(env):
     url = "https://ci.appveyor.com/project/{0}/builds/{1}".format(
         env.get("APPVEYOR_REPO_NAME"), env.get("APPVEYOR_BUILD_ID")
     )
+
+    if env.get("APPVEYOR_REPO_PROVIDER") and env.get("APPVEYOR_REPO_PROVIDER") == "github":
+        repository = "https://github.com/{0}.git".format(env.get("APPVEYOR_REPO_NAME"))
```

Comment:
If `APPVEYOR_REPO_NAME` is not there you'll get `None` in your URL. Not sure it's good?

Reply:
Atm, we're supporting only Github as SCM tool, but you're right. We will iterate this point in the future for sure.